paasta_tools/check_chronos_jobs.py

```diff
@@ -138,9 +135,13 @@ def sensu_message_status_for_jobs(service, instance, job_state_pairs):
         )
     else:
         full_job_id = job_state_pairs[0][0].get('name', '[Couldn\'t fetch job name]')
-        state = job_state_pairs[0][1]
-        sensu_status = sensu_event_for_last_run_state(state)
-        output = message_for_status(sensu_status, service, instance, full_job_id)
+        if job_state_pairs[0][0].get('disabled') is True:
```

Comment:
because the job at [0][0] is just a dict pulled from the api, not a ChronosJobConfig